### ISSUE TITLE
promipool.de - Incorrect Blocking - iOS

### ISSUE THREAD

**Issue by danidoedel** [AdguardTeam/AdguardFilters#4176]
***Site***: http://www.promipool.de/
***Description***: Ads not blocked on iOS. Works fine on desktop computer even with mobile view in dev tools.

<details>
<summary>Screenshot</summary>

![image](https://cloud.githubusercontent.com/assets/10995998/22222111/e22eeea8-e1b5-11e6-9701-7ba14cf88429.png)

</details>

***System configuration***

Information | value
--- | ---
Operating system: | iOS 10.2.1
Browser: | Safari
Adguard version: | 1.1.4
Adguard driver (Windows): | -
Simplified filters(iOS) | Off
Adguard DNS: | None
Stealth mode options (Windows) | -

Adguard filter | Version
--- | ---
.English filter | 1.1.7.7
Spyware filter | 1.0.10.37
Social media filter | 1.0.16.69
Annoyances filter | 1.0.9.24
German filter | 1.0.30.62
Mobile ads filter | 1.0.6.79
Safari filter | 1.0.2.19

**Comment by Alex-302** [comment]
It is clean on my iPad.

**Comment by danidoedel** [comment]
@Alex-302 Still there for me. I think maybe that script does generate these ads?

```javascript
<script type="text/javascript">/* 20170130124001 */ (function(w, d, m){
            function UABPge(selector, travel) {
    var p = UABPqe(selector);
    if (!p) {
        return p;
    }
    if (travel != '') {
        for (var i=0; i < travel.length; i++) {
            if (travel.charAt(i) == '^') {
                p= p.parentNode;
            }
            if (travel.charAt(i) == '<') {
                p= p.previousElementSibling;
            }
            if (travel.charAt(i) == '>') {
                p= p.nextElementSibling;
            }
            if (travel.charAt(i) == 'y') {
                p= p.firstChild;
            }
        }
    }
    return p;
}        function UABPCoord1(event, element)  {
    var rect = element.getBoundingClientRect();
    var x = event.clientX - rect.left;
    var y = event.clientY - rect.top;
    return {'x': x, 'y': y };
}

function UABPpos(e) {
    var xP = 0;
    var yP = 0;
    while (e) {
        xP += (e.offsetLeft + e.clientLeft);
        yP += (e.offsetTop + e.clientTop);
        e = e.offsetParent;
    }
    return { x: xP, y: yP };
}
function UABPCoord2(e, elm) {
    var xP = 0;
    var yP = 0;
    var ip = UABPpos(elm);
    if (e.pageX || e.pageY) {
        xP = e.pageX;
        yP = e.pageY;
    }
    if (e.clientX || e.clientY) {
        xP = e.clientX + document.body.scrollLeft + document.documentElement.scrollLeft;
        yP = e.clientY + document.body.scrollTop + document.documentElement.scrollTop;
    }
    xP = xP - ip.x;
    yP = yP - ip.y;
    return {'x': xP, 'y': yP };
}

function UABPcoord(e, elm) {
    if (elm.getBoundingClientRect) {
        return UABPCoord1(e,elm);
    }
    else {
        return UABPCoord2(e, elm);
    }
}

function UABPcevent(obj, fn) {
    if (obj.addEventListener) {
        obj.addEventListener('click', fn, true);
    } else if (obj.attachEvent) {
        obj.attachEvent('onclick', function () {
            return fn.call(obj, window.event);
        });
    }
}
                function UABPInject() {
            if (!w.uabpdl) {
                return;
            }
            if (w.uabInject) return;
            w.uabInject = true;
            

            w.uabPc = {};
                        w.uabAv = {};
            w.uabAvt = {};
            
                                    var wrappersAndAds = [];

                        if (window.addEventListener && !uabpMobile) {
    window.addEventListener('scroll', function(e) {
        var currentObject = wrappersAndAds['SKYRight'];
        var contentFrame = document.querySelector(".site-main[role]");
        
        if (!contentFrame || !currentObject) {
            return;
        }
        
        if (contentFrame.getBoundingClientRect().top < 5) {
            currentObject.style.setProperty("position", "fixed");
            currentObject.style.setProperty("top", "5px");
        } else {
            currentObject.style.setProperty("position", "absolute");
            currentObject.style.setProperty("top", null);
        }
    });
}

                        var s = UABPce('style');
            s.type = 'text/css';
            UABPih(s, '#ufkvPQCbpi {text-align: center;margin-bottom: 19px;}#ufkvPQCbpi img {cursor: pointer;}#ufkvPQCbpi:before {content: "Anzeige";color: #979797;display: block;margin-bottom: 2px;font: bold 10px/10px "Helvetica Neue Condensed", Helvetica, sans-serif;}#GmNkxCfxs {text-align: center;}#GmNkxCfxs img {cursor: pointer;}#GmNkxCfxs:before {content: "Anzeige";color: #979797;display: block;margin-bottom: 2px;font: bold 10px/10px "Helvetica Neue Condensed", Helvetica, sans-serif;}#hfcMhBtb {position: relative;margin-top: 2px;}.BVYwXX {margin-top: 18px;}.BVYwXX img {cursor: pointer;}#YjhEwVP {position: absolute;margin-left: 792px;}#YjhEwVP img {cursor: pointer;}#UqQuTtql {}#UqQuTtql img {cursor: pointer;}#UqQuTtql:before {content: "Anzeige";color: #979797;display: block;margin-bottom: 2px;font: bold 10px/10px "Helvetica Neue Condensed", Helvetica, sans-serif;}IUuEbQN {}IUuEbQN img {cursor: pointer;}#FVYEkPV {margin-left: 229px;}jhkFoVyd {}jhkFoVyd img {cursor: pointer;}AvnKIMFYQ {}AvnKIMFYQ img {cursor: pointer;}gMgxvbRZc {}gMgxvbRZc img {cursor: pointer;}@media print {#hfcMhBtb,IUuEbQN,#FVYEkPV,jhkFoVyd,AvnKIMFYQ,gMgxvbRZc,#ufkvPQCbpi,#GmNkxCfxs,.BVYwXX,#YjhEwVP,#UqQuTtql{display:none;}} .uabpHid {display:none;}');
            UABPac(h, s);
            
                        var wrap = [];
                                                            var e = UABPge('[role=main]', '');
                                if (e) {
                    var p = UABPce('div');
                    wrap['Wrapper'] = p;
                    wrappersAndAds['Wrapper'] = p;
                                            UABPsi(p, 'hfcMhBtb');
                                                                                                                                                                                                                UABPib(e,p);
                                                                w.uabpd4['w'][2203] = p;
                                            var hb = UABPce('div');
                        UABPih(hb, '<div> </div> <div> </div>');
                        UABPe(hb.childNodes, function(i, o) {
                            UABPib(p, o);
                        });
                    
                    
                    

                }                                                                            var e = UABPge('#ads-outer', '');
                                if (e) {
                    var p = UABPce('span');
                    wrap['BILLWrapperFake'] = p;
                    wrappersAndAds['BILLWrapperFake'] = p;
                                                                                                                                                                                                                if (e.nextSibling) {
                                UABPib(e.nextSibling, p);
                            }
                            else {
                                UABPac(e.parentNode,p);
                            }
                                                                w.uabpd4['w'][2524] = p;
                                            var hb = UABPce('div');
                        UABPih(hb, '<span> </span> <span> </span> <cpan> </cpan>');
                        UABPe(hb.childNodes, function(i, o) {
                            UABPib(p, o);
                        });
                    
                    
                    

                }                                                                            var e = UABPge('.site-main[role]', '');
                                if (e) {
                    var p = UABPce('div');
                    wrap['SKYRightWrapper'] = p;
                    wrappersAndAds['SKYRightWrapper'] = p;
                                            UABPsi(p, 'FVYEkPV');
                                                                                                                                                                                                                if (e.firstChild) {
                                UABPib(e.firstChild, p);
                            }
                            else {
                                UABPac(e,p);
                            }
                                                                w.uabpd4['w'][2559] = p;
                    
                    
                    

                }                                                                            var e = UABPge('.site-content', '');
                                if (e) {
                    var p = UABPce('div');
                    wrap['WrapperFakeSiteContent'] = p;
                    wrappersAndAds['WrapperFakeSiteContent'] = p;
                                                                                                                                                                                                                if (e.nextSibling) {
                                UABPib(e.nextSibling, p);
                            }
                            else {
                                UABPac(e.parentNode,p);
                            }
                                                                w.uabpd4['w'][2560] = p;
                                            var hb = UABPce('div');
                        UABPih(hb, '<div> </div> <div> </div> <div> </div>');
                        UABPe(hb.childNodes, function(i, o) {
                            UABPib(p, o);
                        });
                    
                    
                    

                }                                                                            var e = UABPge('.navigation', '');
                                if (e) {
                    var p = UABPce('div');
                    wrap['WrapperFakeNavigation'] = p;
                    wrappersAndAds['WrapperFakeNavigation'] = p;
                                                                                                                                                                                                                if (e.nextSibling) {
                                UABPib(e.nextSibling, p);
                            }
                            else {
                                UABPac(e.parentNode,p);
                            }
                                                                w.uabpd4['w'][2561] = p;
                                            var hb = UABPce('div');
                        UABPih(hb, '<div> </div> <div> </div> <div> </div> <div> </div>');
                        UABPe(hb.childNodes, function(i, o) {
                            UABPib(p, o);
                        });
                    
                    
                    

                }                                                                            var e = UABPge('#ads-outer', '');
                                if (e) {
                    var p = UABPce('div');
                    wrap['WrapperFakeAdsOuter'] = p;
                    wrappersAndAds['WrapperFakeAdsOuter'] = p;
                                                                                                                                                                                                                if (e.nextSibling) {
                                UABPib(e.nextSibling, p);
                            }
                            else {
                                UABPac(e.parentNode,p);
                            }
                                                                w.uabpd4['w'][2731] = p;
                                            var hb = UABPce('div');
                        UABPih(hb, '<div> </div> <div> </div> <div> </div> <span> </span>');
                        UABPe(hb.childNodes, function(i, o) {
                            UABPib(p, o);
                        });
                    
                    
                    

                }                            
            
            w.uabpPtl = 5;
            w.uabpPl = 0;
                                                                            var e2201 = UABPge('#ad-fullbanner2-billboard-outer', '');
                                                if (e2201) {
                            UABPi(uabpCy('2xjo|2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2odeho0mxol0whuplq0}dkq0'+w.uabpRnd+'24;69895040jhu0GH2rsd0zdqq0hiihfwv0xjo|b4bwkxpeqdlo1msj',-3), function() {
                                        var i = this;
                                        if (i.height < 2) {
                                            return;
                                        }
                                        UABPpd();
                                        var e = e2201;
                                        var p = UABPce('div');
                                        wrappersAndAds['BILL'] = p;
                                                                                                                            UABPsi(p, 'ufkvPQCbpi');
                                                                                                                            UABPsc(p, 'RpRUbZwva ');
                                        

                                        var ie = i;
                                        
                                        UABPcevent(ie, function(e,p) {
                                            if (w.uabPc[ 2201 ] ) {
                                                
                                                return;
                                            }
                                            
                                            e.preventDefault();
                                            e.stopPropagation();
                                            var p = UABPcoord(e, this);
                                            
                                            
                                            var cus = [uabpCy('2txrwh2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2ndvvh0zddjhq0ndvvh0vdjhq0'+w.uabpRnd+'2<<<7537040jhu0GH2lqn0mhuu|0|dfnb4bwkxpeqdlo1msj',-3),uabpCy('2{hur{hg2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2zdko0kdihq0{ly0'+w.uabpRnd+'2955;757040jhu0GH2glhvh0xjo|0zdoo0udglxvb4bwkxpeqdlo1msj',-3),uabpCy('2hjjv2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2gxiw0ndqq0dejdeh0ydwhu0{hur{0'+w.uabpRnd+'25<<3:4<040jhu0GH2zdqq0pdfkh0nlzl0qljkwvb4bwkxpeqdlo1msj',-3),uabpCy('2odwh{2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2dejdeh0ohkuhu0wdj0kdihq0'+w.uabpRnd+'28;6;67;040jhu0GH2wrg0zdoosdshu0vdfkhq0udkphq0xpihogb4bwkxpeqdlo1msj',-3),];
                                            
                                                                                            var cu = cus[ Math.floor(p.x * 4 / this.offsetWidth) ];
                                                UABPop(w, cu, (e.button == 0 && e.ctrlKey) ? '_blank' : '_self');
                                                                                    });

                                        

                                                                                    UABPac(p,i);
                                        
                                        
                                                                                                                                    UABPib(e,p);
                                                                                                                            w.uabpd4['a'][2201] = p;
                                                                                                                                                                                                            var hb = UABPce('div');
                                            UABPih(hb, '<div> </div> <div> </div> <div> </div> <div> </div>');
                                            UABPe(hb.childNodes, function(i, o) {
                                                UABPib(p, o);
                                            });
                                                                                    
                                                                                UABPvc(p, 2201, uabpCy('2wdohqw2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2zhj0ndqq0ehl0kddu0dejdeh0'+w.uabpRnd+'2465;65;040jhu0GH2udfhu0vdloru0fodlu0jdehb4bwkxpeqdlo1msj',-3));                                                                    }, UABPpd);
                        } else {
                            UABPpd();
                                                    }
                                                                                            var e2202 = UABPge('[role=main]', '');
                                                if (e2202) {
                            UABPi(uabpCy('2ioduh2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2lfk0lghh0eoxph0'+w.uabpRnd+'27:<884:040jhu0GH2mdsdq0|xsslh0txdunb4bwkxpeqdlo1msj',-3), function() {
                                        var i = this;
                                        if (i.height < 2) {
                                            return;
                                        }
                                        UABPpd();
                                        var e = e2202;
                                        var p = UABPce('div');
                                        wrappersAndAds['BILLBottom'] = p;
                                                                                                                            UABPsi(p, 'GmNkxCfxs');
                                                                                                                            UABPsc(p, 'CDVstytJ ');
                                        

                                        var ie = i;
                                        
                                        UABPcevent(ie, function(e,p) {
                                            if (w.uabPc[ 2202 ] ) {
                                                
                                                return;
                                            }
                                            
                                            e.preventDefault();
                                            e.stopPropagation();
                                            var p = UABPcoord(e, this);
                                            
                                            
                                            var cus = [uabpCy('2mhuu|2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2qdhjho0slov0xvd0'+w.uabpRnd+'293<33;5040jhu0GH2shhuv0wdfnoh0jherwh0lqghhgb4bwkxpeqdlo1msj',-3),uabpCy('2khdgkxqwhu2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2qhkph0udkphq0sidqg0gxiw0rihq0'+w.uabpRnd+'27897<73040jhu0GH2kdihq0kddu0lqvhoq0grru0hlqhb4bwkxpeqdlo1msj',-3),uabpCy('2gholjkw2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2xpihog0ohlp0xuzdog0gdwhl0odwhlq0'+w.uabpRnd+'253:576;040jhu0GH2phdo0de}xj0txdunb4bwkxpeqdlo1msj',-3),uabpCy('2jkrvw2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2hlfkh0zhj0zdqq0'+w.uabpRnd+'2793449;040jhu0GH2hfnh0pdwhuldo0sdfnhqb4bwkxpeqdlo1msj',-3),];
                                            
                                                                                            var cu = cus[ Math.floor(p.x * 4 / this.offsetWidth) ];
                                                UABPop(w, cu, (e.button == 0 && e.ctrlKey) ? '_blank' : '_self');
                                                                                    });

                                        

                                                                                    UABPac(p,i);
                                        
                                        
                                                                                                                                    UABPac(e,p);
                                                                                                                            w.uabpd4['a'][2202] = p;
                                                                                                                                                                                                            var hb = UABPce('div');
                                            UABPih(hb, '<div> </div> <div> </div> <div> </div> <div> </div>');
                                            UABPe(hb.childNodes, function(i, o) {
                                                UABPib(p, o);
                                            });
                                                                                    
                                                                                UABPvc(p, 2202, uabpCy('2edfnorjv2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2}dxq0}lhkw0lqvhoq0mdsdq0'+w.uabpRnd+'2;8<54:8040jhu0GH2}rqh0lghh0odph0}lsb4bwkxpeqdlo1msj',-3));                                                                    }, UABPpd);
                        } else {
                            UABPpd();
                                                    }
                                                                                            var e2204 = wrap['Wrapper'];
                                                if (e2204) {
                            UABPi(uabpCy('2lpsuryh2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2vdjhq0xvd0dvshnw0'+w.uabpRnd+'248:6873040jhu0GH2odfnhq0zdjrq0de}xj0udfhub4bwkxpeqdlo1msj',-3), function() {
                                        var i = this;
                                        if (i.height < 2) {
                                            return;
                                        }
                                        UABPpd();
                                        var e = e2204;
                                        var p = UABPce('div');
                                        wrappersAndAds['LBTop'] = p;
                                                                                                                                                                    UABPsc(p, 'BVYwXX ');
                                        

                                        var ie = i;
                                        
                                        UABPcevent(ie, function(e,p) {
                                            if (w.uabPc[ 2204 ] ) {
                                                
                                                return;
                                            }
                                            
                                            e.preventDefault();
                                            e.stopPropagation();
                                            var p = UABPcoord(e, this);
                                            
                                            
                                            var cus = [uabpCy('2txdun2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2odeho0vhjhoq0xuzdog0kddu0'+w.uabpRnd+'24<3;38<040jhu0GH2odwhlq0udoo|0wdfnohb4bwkxpeqdlo1msj',-3),uabpCy('2hiihfwv2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2ilupd0dejdeh0frxs0'+w.uabpRnd+'2;4<96;7040jhu0GH2}hurv0mhpdqg0whdub4bwkxpeqdlo1msj',-3),uabpCy('2|dfn2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2sdduh0udkphq0jhehu0vhjhoq0'+w.uabpRnd+'29978:39040jhu0GH2nlfn0khpg0ilohv0phgldb4bwkxpeqdlo1msj',-3),];
                                            
                                                                                            var cu = cus[ Math.floor(p.x * 3 / this.offsetWidth) ];
                                                UABPop(w, cu, (e.button == 0 && e.ctrlKey) ? '_blank' : '_self');
                                                                                    });

                                        

                                                                                    UABPac(p,i);
                                        
                                        
                                                                                                                                    UABPac(e,p);
                                                                                                                            w.uabpd4['a'][2204] = p;
                                                                                
                                                                                UABPvc(p, 2204, uabpCy('2kdugo|2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2sdfnhq0lfk0wxpru0'+w.uabpRnd+'259685;:040jhu0GH2udoo|0}dkohq0idq0hiihnw0mrevb4bwkxpeqdlo1msj',-3));                                                                    }, UABPpd);
                        } else {
                            UABPpd();
                                                    }
                                                                                            var e2205 = wrap['SKYRightWrapper'];
                                                if (e2205) {
                            UABPi(uabpCy('2nlzl2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2|dfkw0ohlp0hkh0mxol0jhehu0'+w.uabpRnd+'2:<<3<;<040jhu0GH2gdehl0hpedujr0mhpdqg0edfnolvwb4bwkxpeqdlo1msj',-3), function() {
                                        var i = this;
                                        if (i.height < 2) {
                                            return;
                                        }
                                        UABPpd();
                                        var e = e2205;
                                        var p = UABPce('div');
                                        wrappersAndAds['SKYRight'] = p;
                                                                                                                            UABPsi(p, 'YjhEwVP');
                                                                                                                                                                                                                                                                

                                        var ie = i;
                                        
                                        UABPcevent(ie, function(e,p) {
                                            if (w.uabPc[ 2205 ] ) {
                                                
                                                return;
                                            }
                                            
                                            e.preventDefault();
                                            e.stopPropagation();
                                            var p = UABPcoord(e, this);
                                            
                                            
                                            var cus = [uabpCy('2qhyhu2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2xuzdog0mhpdqg0klq0dvshnw0'+w.uabpRnd+'26894577040jhu0GH2idoo0pdfkh0edfnxs0ydoxh0{hulfb4bwkxpeqdlo1msj',-3),uabpCy('2gudj2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2gxiw0mhpdqg0eoxph0'+w.uabpRnd+'24:;4:<;040jhu0GH2kdihq0grru0{ly0pdvvhq0yhueb4bwkxpeqdlo1msj',-3),uabpCy('2khfwlf2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2vhjhoq0klq0kdihq0gdehl0'+w.uabpRnd+'29437<8:040jhu0GH2pdjhq0riilfh0{hur{hg0hfnh0}lhkwb4bwkxpeqdlo1msj',-3),];
                                            
                                                                                            var cu = cus[ Math.floor(p.y * 3 / this.offsetHeight) ];
                                                UABPop(w, cu, (e.button == 0 && e.ctrlKey) ? '_blank' : '_self');
                                                                                    });

                                        

                                                                                    UABPac(p,i);
                                        
                                        
                                                                                                                                    UABPac(e,p);
                                                                                                                            w.uabpd4['a'][2205] = p;
                                                                                
                                                                                UABPvc(p, 2205, uabpCy('2mdfn2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2qdph0urfn0qdfkw0'+w.uabpRnd+'2<9:;486040jhu0GH2edfnvwdjh0grp0zdiih0wxprub4bwkxpeqdlo1msj',-3));                                                                    }, UABPpd);
                        } else {
                            UABPpd();
                                                    }
                                                                                            var e2206 = UABPge('.gtm-ad', '');
                                                if (e2206) {
                            UABPi(uabpCy('2dffodlp2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2zdqq0ohlp0xqvhu0}lhkw0fdih0'+w.uabpRnd+'2<679384040jhu0GH2pdvvhq0gdehl0revw0udkphqb4bwkxpeqdlo1msj',-3), function() {
                                        var i = this;
                                        if (i.height < 2) {
                                            return;
                                        }
                                        UABPpd();
                                        var e = e2206;
                                        var p = UABPce('div');
                                        wrappersAndAds['ARTENDM'] = p;
                                                                                                                            UABPsi(p, 'UqQuTtql');
                                                                                                                                                                                                                                                                

                                        var ie = i;
                                        
                                        UABPcevent(ie, function(e,p) {
                                            if (w.uabPc[ 2206 ] ) {
                                                
                                                return;
                                            }
                                            
                                            e.preventDefault();
                                            e.stopPropagation();
                                            var p = UABPcoord(e, this);
                                            
                                            
                                            var cus = [uabpCy('2xpeuhood2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2jhehu0{hur{0vhjhoq0'+w.uabpRnd+'28;;4776040jhu0GH2kdss|0wrix0rkqh0sdg0kdss|b4bwkxpeqdlo1msj',-3),uabpCy('2mdlohg2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2slov0sdfnhq0idoo0lghh0'+w.uabpRnd+'256;6:44040jhu0GH2wxpru0phgld0xqvhub4bwkxpeqdlo1msj',-3),uabpCy('2txdnh2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2jherwh0kdihq0|rjd0hlqh0'+w.uabpRnd+'28478<58040jhu0GH2khug0zdiih0wkhuhb4bwkxpeqdlo1msj',-3),];
                                            
                                                                                            var cu = cus[ Math.floor(p.x * 3 / this.offsetWidth) ];
                                                UABPop(w, cu, (e.button == 0 && e.ctrlKey) ? '_blank' : '_self');
                                                                                    });

                                        

                                                                                    UABPac(p,i);
                                        
                                        
                                                                                                                                    UABPib(e,p);
                                                                                                                            w.uabpd4['a'][2206] = p;
                                                                                                                                                                
                                                                                UABPvc(p, 2206, uabpCy('2ioduh2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2zhj0ndvvh0zdko0pdxuhu0odfnhq0'+w.uabpRnd+'2;<:;644040jhu0GH2udfhu0revw0yhue0ydwhu0xuzdogb4bwkxpeqdlo1msj',-3));                                                                    }, UABPpd);
                        } else {
                            UABPpd();
                                                    }
                            
                                }

        function UABPpd() {
            w.uabpPl++;
                    }



            
        function UABPTrack() {
        if (w.uabTrack) return false;
        w.uabTrack = true;
        

        
        UABPInject();
                var f = function() {
            if (!UABPIBlocked(this)) {
                
            }        }
        UABPi(uabpCy('2mhuu|2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2gxiw0zdqq0urfn0'+w.uabpRnd+'28697985040jhu0GH2wrg0wdohqw0mdsdq0lqvhoq0frxsb4bwkxpeqdlo1msj',-3),f,f);
    }

        function UABPdd() {
        w.uabpd3++;
        
                    if (w.uabpd3 == w.uabpd1.length) {
                                if (w.uabpd2.length == w.uabpd3) {
                    w.uabpij = true;
                    UABPTrack();
                }
            }
        
    }
    w.UABPdd = UABPdd;

        function UABPgcs(o, p) {
        if (window.getComputedStyle) {
            return window.getComputedStyle(o, null).getPropertyValue(p);
        }
    }

        function UABPvis(o) {
        var d = UABPgcs(o, 'display');
                if (d && d == 'none') return false;

        var d = UABPgcs(o, 'visibility');
                if (d && d == 'hidden') return false;
        return true;
    }

        function UABPifd(o) {
        return o.contentDocument || o.contentWindow.document;
    }

        function UABPifw(o) {
        return o.contentWindow || o;
    }

    function UABPDetect() {
                UABPDoDetect();
        
    }
    function UABPDoDetect() {
        if (w.uabDetect) return;
        w.uabDetect = true;
        


        w.uabpd1 = []; // detection functions
        w.uabpd2 = []; // elements of blocked detections
        w.uabpd3 = 0;  // number of detections finished
        w.uabpd4 = {}; // injected ads and wrappers
        w.uabpd4['a'] = {};
        w.uabpd4['w'] = {};


                    var d1 = UABPce('DIV');
            UABPih(d1, '.');
            UABPac(b, d1);

            var d2 = UABPce('DIV');
            var d2p = d2;
                            var p = d2, c = d2;
                                                    UABPsc(c, 'wingadblock');                                            UABPih(c, '.');
                                                                                UABPac(b, d2);
            d2 = c;
        
        var f = function() {
            if (!UABPIBlocked(this)) {
                
                var ua = navigator.userAgent || navigator.vendor || window.opera;
                w.uabpMobile = ua.match( /iPad/i ) || ua.match( /iPhone/i ) || ua.match( /iPod/i ) || ua.match( /Android/i );

                    var fc = function() {
        setTimeout(function(){
            if (UABPEBlocked(d1, d2)) {
                
                w.uabpd2.push(d2);
            }            UABPre(d1);
            UABPre(d2p);
            UABPdd();
        }, 500);
    }
    w.uabpd1.push(fc);
                    var f = function(i) {
        var ab = UABPIBlocked(this);
        if (ab) {
            w.uabpd2.push(this);
            
        }        UABPdd();
    }
            var fc = function() {
            UABPi(uabpCy('2odqh2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2dgberpe2rkqh0ihghu0kdihq0'+w.uabpRnd+'286::664040jhu0GH2|hvwhugd|0}rqh0sdqho0lpsdfwv0pdjhqb4bwkxpeqdlo1sqj',-3), f, f);
        }
        w.uabpd1.push(fc);
                                                    
                for (var i = 0; i < w.uabpd1.length; i++) {
                    w.uabpd1[i]();
                }
            }        }
        UABPi(uabpCy('2dffhohudwru2surplsrro2vwrudjh2lpdjhv2phgld2lpdjhv2fkhiv0ndehoq0dejdeh0yhuerw0'+w.uabpRnd+'24;5<68:040jhu0GH2xvd0nruhd0edvlfb4bwkxpeqdlo1msj',-3),f,f);
    }

    function UABPInit() {
        if (w.uabpdl) return;
        w.uabpdl = true;
        var ix = setInterval(function() {
            b = UABPqe('body');
            if (b != null) {
                if (w.uabBody) return;
                w.uabBody = true;
                clearInterval(ix);
                
                                                            w.uabpRnd = UABPRnd();
                        UABPDetect();
                    
                            }        }, 10);
    }

    
        function UABPRnd() {
        var r = UABPRndCln();
        
        return uabpCy(r,3);
    }
    function UABPRndCln() {
        var c = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
        var l = m.max(8,1+m.floor((m.random() * 8)));
        var r = '';
        while (r.length < l) {
            r += c.charAt( m.floor((m.random() * c.length)));
        }
        return r;
    }
    function UABPce(n) {
    return d.createElement(n);
}
function UABPih(e,c) {
    e.innerHTML = c;
}
function UABPac(e,c) {
    e.appendChild(c);
}
function UABPsb(e,i) {
    e.style.backgroundImage="url(" + i.src  +")";
}
function UABPib(e,c) {
    e.parentElement.insertBefore(c,e);
}
function UABPre(e) {
    if (e && e.parentElement) {
        e.parentElement.removeChild(e);
    }
}

function UABPhc(e, c) {
    return new RegExp(' ' + c + ' ').test(' ' + e.className + ' ');
}

function UABPsc(e,c) {
    e.className = c;
}

function UABPrc(e, c) {
    var nc = ' ' + e.className.replace( /[\t\r\n]/g, ' ') + ' ';
    if (UABPhc(e, c)) {
        while (nc.indexOf(' ' + c + ' ') >= 0 ) {
            nc = nc.replace(' ' + c + ' ', ' ');
        }
        e.className = nc.replace(/^\s+|\s+$/g, '');
        if (e.className == '') {
            e.removeAttribute("class");
        }
    }
}

function UABPsi(e,i) {
    e.id = i;
}
function UABPsh(e,i) {
e.href = i;
}
function UABPqe(s) {
    return d.querySelector(s);
}
function UABPqes(s) {
    return d.querySelectorAll(s);
}
function UABPi(u,l,e) {
    var i = UABPce('IMG');
    i.onload = l;
    i.onerror = e;
    i.src = u;
}

function UABPe(e,c) {
    if (e && e.length) {
        for (var i = 0; i < e.length; i++) {
            c(i,e[i]);
        }
    }
}

function UABPif(w) {
    try {
        return w.self !== w.top;
    } catch (e) {
        return true;
    }
}
function UABPop(w, u, t) {
    if (t === '_blank' || !UABPif(w)) {
        w.open(u,t);
    }
    w.top.open(u,t);
}        function UABPIBlocked(i) {
        
        return i.height != 2;
    }

    function UABPEBlocked(o,t) {
        
        
        return (t.offsetHeight==0 && o.offsetHeight != 0)
    }
    function UABPIEBlocked(i) {
        
        return (i.offsetHeight==0)
    }    function uabpCy(s, o) {
            var r = '';
        for (i = 0; i < s.length; i++) {
            r += String.fromCharCode(s.charCodeAt(i) + o);
        }
        return r.replace('\\\\', '\\\\\\\\');
    }        function UABPvc(e, i, u) {
    if (!w.uabAv[i]) {
        w.uabAv[i]=0;
        w.uabAvt[i] = w.setTimeout(function(){ UABPvc(e, i, u); }, 100);
    }
    if (w.uabAv[i]>=2) return;
    var er = e.getBoundingClientRect();
    if( typeof(e.width) === "undefined") {
        var h = e.offsetHeight;
        var wi = e.offsetWidth;
    } else {
        var h = e.height;
        var wi = e.width;
    }
    if (h == 0 || wi == 0) {
        
        return;
    }
    var w2 = er.left + wi / 2;
    var h2 = er.top + h / 2;
    var de = d.documentElement;
    var ash = 0; var asw = 0;
    if(w.innerWidth && w.innerHeight)  {
        ash = w.innerHeight;
        asw = w.innerWidth;
    } else if(de && !isNaN(de.clientHeight) && !isNaN(de.clientWidth)) {
        ash = d.clientHeight;
        asw = d.clientWidth;
    }
    if(0 <= h2 && h2 <= ash && 0 <= w2 && w2 <= asw) {
        w.uabAv[i]++;
        if(w.uabAv[i]>=200/100) {
            var x = function() {};
            UABPi(u, x, x)
            
            w.clearTimeout(w.uabAvt[i]);
        }
    } else {
        w.uabAv[i]=0;
    }
}    
    var h = UABPqe('head');
    var b = null;
    d.addEventListener ? d.addEventListener("DOMContentLoaded", UABPInit, false) : d.attachEvent && d.attachEvent("onreadystatechange", function () { "complete" === d.readyState && UABPInit(); });
    w.addEventListener ? w.addEventListener("load", UABPInit, false) : w.attachEvent && w.attachEvent("onload", UABPInit);


    
})(window, document, Math)
 /* 52f7ba62e18c18044fabec01150bb8a2 */</script>
```

**Comment by danidoedel** [comment]
@Alex-302 That rule does fix it for me:
`@@/ad_bomb/*$image,domain=promipool.de`

Same for the site http://pc-magazin.de: `@@/ad_bomb/*$image,domain=pc-magazin.de` (let me know if you want me to open a separate issue for this)

**Comment by Alex-302** [comment]
Ok, added
`@@/ad_bomb/*$image,domain=promipool.de|pc-magazin.de`
